script/tool/lib/src/publish_plugin_command.dart

```diff
@@ -408,15 +403,15 @@ Safe to ignore if the package is deleted in this commit.
     return statusOutput.isEmpty;
   }
 
-  Future<String> _verifyRemote(String remote) async {
+  Future<String?> _verifyRemote(String remote) async {
     final io.ProcessResult remoteInfo = await processRunner.run(
```

Comment:
nits: Maybe have a different name here as there is a `RemoteInfo` class now and it might be confusing.
maybe getRemoteUrlResult?

Reply:
Done.